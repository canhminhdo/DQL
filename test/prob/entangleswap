#!/bin/sh

DIR=../../test/prob
cd ../../examples/prob && maude < entangleswap.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/entangleswap.out
diff $DIR/entangleswap.expected $DIR/entangleswap.out
rm $DIR/entangleswap.out