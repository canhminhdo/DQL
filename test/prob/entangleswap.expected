==========================================
reduce in ENTAGLE-SWAP : initQState |= [H(0) ; CX(0,1) ; H(2) ; CX(2,3) ; CX(1,
    2) ; H(1) ; ((P(2,|0>) ? with prob == 1/2) ; skip U (P(2,|1>) ? with prob
    == 1/2) ; X(3)) ; ((P(1,|0>) ? with prob == 1/2) ; skip U (P(1,|1>) ? with
    prob == 1/2) ; Z(3)) ; SWAP(1,3)]P(0,1,EPR) .
result JudgmentSet: emptyJS
Bye.
